app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy > 0:
+        return f"Friends should buy {masks_to_buy} masks"
```

Comment:
This `if` condition is unnecessary. You can directly return the string with the number of masks to buy. If `masks_to_buy` is `0`, it will simply say that friends should buy 0 masks, which is fine and makes the code simpler.